worldedit-core/src/main/java/com/sk89q/worldedit/internal/cui/SelectionPoint2DEvent.java

```diff
@@ -27,22 +27,38 @@
     protected final int id;
     protected final int blockX;
     protected final int blockZ;
-    protected final int area;
+    protected final long area;
 
+    @Deprecated
     public SelectionPoint2DEvent(int id, BlockVector2 pos, int area) {
```

Comment:
The old code still works fine with these gone, but I fear that it'll break binary compatibility if I do. Am I correct in assuming this?